lib/oli/delivery/sections/browse.ex

```diff
@@ -7,6 +7,8 @@ defmodule Oli.Delivery.Sections.Browse do
   alias Oli.Repo
   alias Oli.Repo.{Paging, Sorting}
 
+  @chars_to_replace_on_search [" ", "&", ":", ";", "(", ")", "|", "!", "'", "<", ">"]
```

Comment:
Should we include more special characters to this list?

Reply:
Is mainly because we shouldn't collide with the characters [postgres text search](https://www.postgresql.org/docs/current/datatype-textsearch.html) uses to build the lexemes